src/components/__tests__/autocomplete.test.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { ApolloProvider } from '@apollo/client';
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { client } from '../../ApolloClient';
+import { setupServer } from '../../mocks/setupServer';
+import Autocomplete from '../autocomplete';
+
+const autoCompleteSetup = () => {
+  render(
+    <ApolloProvider client={client}>
+      <Autocomplete />
+    </ApolloProvider>
+  );
+};
+
+describe('Testing Autocomplete w getBreedsByName query', () => {
+  setupServer();
```

Comment:
This gets executed once per `describe`. Most likely, you want to do it before each test. If not, `beforeAll` can be used scoped in the `describe`. Either way, it's more explicit and easier for the reader.